commands/command-template/example-command.js

```diff
@@ -0,0 +1,84 @@
+// Imports the discord.js-commando library which is required for commands to work. 
+const { Command } = require('discord.js-commando');
+
+// Create the embedded message layout for the bots message. 
+// Im still not 100% sure what value the embed color value is from but if/
+// this website is very helpful with generating the needed color and code
+// https://leovoel.github.io/embed-visualizer/ 
+const exampleEmbed = {
+	// This is the main body of the embedded message. The \n character will
+	// add new lines for extra space in the command.
+	"description": "This is the Craftory Bot command template. If you are wanting to help improve the bot learn more here. \n\nNOTE: This command will be removed in 5 seconds! \n\nhttps://github.com/CraftoryStudios/DiscordBot",
+	
+	// Color can be any value and must be a value from this website. https://leovoel.github.io/embed-visualizer/ 
+	// To get the color choose the color you want then click on "Generate Code".
+	// The look for the color value which should be a long number like the one below. 
+	"color": 3077393,
+	
+	// The author is what the smaller icon and name is under.
+	"author": {
+		
+		// name - Is what is reported when the command is issued.
+		//		  The name much be what the command is about. 
+		// For example: Becasue this is the example command the name
+		//				will be Example Command.
```

Comment:
```suggestion
		/* name - Is reported when the command is issued.
		 For example: Because this is the example command the name will be "Example Command". */
```